src/core/Akka.Cluster/ClusterDaemon.cs

```diff
@@ -816,6 +816,8 @@ public ClusterEvent.IClusterDomainEvent Event
     /// </summary>
     internal sealed class ClusterDaemon : ReceiveActor, IRequiresMessageQueue<IUnboundedMessageQueueSemantics>
     {
+        private Cluster _cluster;
```

Comment:
Where does this value get assigned?

Reply:
in `ClusterDaemon .CreateChildren` method
The message `InternalClusterAction.GetClusterCoreRef` acts as the init message from `Cluster` constructor